api/tests/Unit/PoolCandidatePolicyTest.php

```diff
@@ -0,0 +1,281 @@
+            'email' => 'guest-user@test.com',
+            'sub' => 'guest-user@test.com',
+        ]);
+        $this->guestUser->syncRoles(["guest"]);
+
+        $this->applicantUser = User::factory()->create([
+            'email' => 'applicant-user@test.com',
+            'sub' => 'applicant-user@test.com',
+        ]);
+        $this->applicantUser->syncRoles($baseRoles);
+
+        $this->poolOperatorUser = User::factory()->create([
+            'email' => 'pool-operator-user@test.com',
+            'sub' => 'pool-operator-user@test.com',
+        ]);
+        $this->poolOperatorUser->syncRoles($baseRoles);
+        $this->team = Team::factory()->create(['name' => 'test-team']);
+        $this->poolOperatorUser->attachRole("pool_operator", $this->team);
+
+        $this->requestResponderUser = User::factory()->create([
+            'email' => 'request-responder-user@test.com',
+            'sub' => 'request-responder-user@test.com',
+        ]);
+        $this->requestResponderUser->syncRoles([
+            ...$baseRoles,
+            "request_responder"
+        ]);
+
+        $this->adminUser = User::factory()->create([
+            'email' => 'platform-admin-user@test.com',
+            'sub' => 'platform-admin-user@test.com',
+        ]);
+        $this->adminUser->syncRoles([
+            ...$baseRoles,
+            "request_responder",
+            "platform_admin"
+        ]);
+
+        $this->candidateUser = User::factory()->create([
+            'email' => 'candidate-user@test.com',
+            'sub' => 'candidate-user@test.com',
+        ]);
+        $this->candidateUser->syncRoles($baseRoles);
+
+        $this->teamPool = Pool::factory()->create([
+            'user_id' => $this->poolOperatorUser->id,
+            'team_id' => $this->team->id,
+        ]);
+
+        $this->poolCandidate = PoolCandidate::factory()->create([
+            'user_id' => $this->candidateUser->id,
+            'pool_id' => $this->teamPool->id
+        ]);
+    }
+
+
+    /**
+     * Assert that no user may view any pool candidate
+     *
+     * @return void
+     */
+    public function testViewAny()
+    {
+        $this->assertFalse($this->guestUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->applicantUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->candidateUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->poolOperatorUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->requestResponderUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->adminUser->can('viewAny', PoolCandidate::class));
+    }
+
+    /**
+     * Assert that only owner can view draft pool candidates
+     *
+     * @return void
+     */
+    public function testViewDraft()
+    {
+        $this->poolCandidate->submitted_at = null;
+        $this->poolCandidate->pool_candidate_status = ApiEnums::CANDIDATE_STATUS_DRAFT;
```

Comment:
Use submitted_at to control draft status now.

Reply:
Updated in [improve policy checks and make test more strict](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/594bfd83547582f3b67a6967241f4ebacd3b18f0)